### PR TITLE
Adding an easyconfig for cellranger-atac

### PR DESCRIPTION
**Pull request by rtripath89**

### CODE REVIEW

File: easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-1.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+name = "CellRanger-ATAC"
+version = "1.2.0"
+
+homepage = "https://support.10xgenomics.com/single-cell-atac/software/pipelines/latest/what-is-cell-ranger-atac"
+description = """Cell Ranger ATAC is a set of analysis pipelines that process 
+Chromium Single Cell ATAC data."""
+
+toolchain = SYSTEM
+
+# Download manually from https://cf.10xgenomics.com/releases/cell-atac/cellranger-atac-1.2.0.tar.gz
```
Comment (zao) [inline easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-1.2.0.eb:16]: Visiting this URL results in an XML error about missing cookies, navigating via `homepage` ends up on a mandatory registration page.

Comment (rtripath89) [reply easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-1.2.0.eb]: I have changed the commented out link to point to homepage of the software. you should be able to use the wget or curl command provided on that page to download it directly on a system.

Comment (zao) [reply easybuild/easyconfigs/c/CellRanger-ATAC/CellRanger-ATAC-1.2.0.eb]: Can't test this as I don't want to register for a software that has license restrictions. I guess it's as good as it gets.

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node254.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz (broadwell), Python 2.7.5
See https://gist.github.com/2eb8a7365e51e4a8c625ce7e6874bea2 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @rtripath89!

**Pull request merged**